js/scripts.js

```diff
@@ -0,0 +1,8 @@
+var a = 5;
+var b = 6;
+var value = (a * a) + (2 * a * b) - (b * b);
+
+(value === 0) ? alert('wynik dziłania: (a * a) + (2 * a * b) - (b * b) równa się "0"') : alert('wynik dziłania: (a * a) + (2 * a * b) - (b * b) nie jest równy "0"')
```

Comment:
nie do końca się zrozumieliśmy.

Chciałbym żebyś sprawdził CZY liczba jest równa zero, jeśli tak to alert, jeśli nie to ternary, który zadecyduje czy alert powiadomi o tym, że liczba jest dodatnia, czy ujemna